pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -3252,10 +3213,6 @@ in
     }
   );
 
-  ssr = super.ssr-nvim.overrideAttrs {
```

Comment:
Believe this should be re-added and just fix the package name. Probably need to add an alias, in case someone has been relying on this bug.

Reply:
Looks like it's been like this since it was initially added in https://github.com/NixOS/nixpkgs/pull/202739 (ed360a1c0a9f531b141175c31c7a881a0f29bc5f).

Which name should be the canonical one? `ssr-nvim`?

Reply:
yeah, nixpkgs will just convert `.` to `-`